src/components/search/SearchTextInput.tsx

```diff
@@ -0,0 +1,21 @@
+import { TextField } from '@mui/material'
+import { Fragment } from 'react'
+import SimpleSearchTextInput from '../../types/SimpleSearchTextInput'
```

Comment:
Let's declare props type in the same file.

Reply:
Yes. Thank you